web/app/page.js

```diff
@@ -40,14 +32,20 @@ export default function Home() {
           {allergens.map(({ name, time }) => {
             return (
               <li key={name}>
-                <p className="font-bold text-lg">{name}</p>
-                <Since props={time} />
-                <Resetter
-                  props={{
-                    updater: updateTime,
-                    name,
+                <p
+                  className="font-bold text-lg"
+                  suppressHydrationWarning={true}
```

Comment:
for another PR, but I'd like to see if we can remove use of this where possible. I could be wrong but I suspect that more often than not it's a code smell (hint that something should be fixed)

Reply:
I think the cleaner solution is disabling server side pre rendering, as detailed [in the Next.js docs.](https://nextjs.org/docs/messages/react-hydration-error) It might require restructuring the app, but I'll give it a shot in another PR. They specifically call out timestamps as a valid use of suppressHydrationWarning however.